pkg/storegateway/series_chunks_test.go

```diff
@@ -693,14 +764,20 @@ func TestLoadingSeriesChunksSetIterator(t *testing.T) {
 			}
 
 			// Run test
-			set := newLoadingSeriesChunksSetIterator(*readers, newSliceSeriesChunkRefsSetIterator(nil, testCase.setsToLoad...), 100, newSafeQueryStats(), minT, maxT)
+			set := newLoadingSeriesChunksSetIterator(*readers, newSliceSeriesChunkRefsSetIterator(nil, testCase.setsToLoad...), 100, newSafeQueryStats(), minT, maxT, testCase.ignoreNativeHistograms)
 			loadedSets := readAllSeriesChunksSets(set)
 
 			// Assertions
 			if testCase.expectedErr != "" {
 				assert.ErrorContains(t, set.Err(), testCase.expectedErr)
 			} else {
 				assert.NoError(t, set.Err())
+
+				if testCase.expectNoChunkBytesSlicePoolLoads {
+					assert.Zero(t, chunkBytesSlicePool.(*pool.TrackedPool).Gets.Load())
+				} else {
+					assert.Greater(t, chunkBytesSlicePool.(*pool.TrackedPool).Gets.Load(), int64(0))
+				}
```

Comment:
What's the reason for adding this assertion here as well?

Reply:
Honestly, I can't think of a good reason - removed!